.circleci/config.yml

```diff
@@ -144,7 +144,7 @@ jobs:
       - run:
           name: Run slow tests
           working_directory: ~/repo/plasma_framework/python_tests
-          no_output_timeout: 10h
+          no_output_timeout: 5h # Circl CI limits to 5hr max: https://discuss.circleci.com/t/job-times-out-after-5-hours/32220/3
```

Comment:
Actually, this setting has little to do with the job timeout itself, since each iteration prints sth to stdout.

Reply:
ah, you're right. I remove the `no_output_timeout` to not confuse anymore.

Reply:
okay...that is a bad decision to remove it totally as 10 minutes defaults seems not enough 

https://app.circleci.com/pipelines/github/omisego/plasma-contracts/2031/workflows/472e1621-cfea-4290-9be3-b323b51d92dd/jobs/10568